adserver/templates/adserver/publisher/overview.html

```diff
@@ -0,0 +1,78 @@
+{% extends "adserver/base.html" %}
+{% load crispy_forms_tags %}
+{% load humanize %}
+{% load i18n %}
+
+
+{% block title %}{% trans 'Publisher overview' %} - {{ publisher }}{% endblock %}
+
+
+{% block breadcrumbs %}
+  {{ block.super }}
+
+  {% url 'publisher_main' publisher.slug as publisher_main_url %}
+  {% if request.path == publisher_main_url %}
+    <li class="breadcrumb-item active">{{ publisher }}</li>
+  {% else %}
+    <li class="breadcrumb-item"><a href="{{ publisher_main_url }}">{{ publisher }}</a></li>
+  {% endif %}
+
+{% endblock breadcrumbs %}
+
+
+{% block content_container %}
+  <section>
+    <h1>{% blocktrans %}{{ publisher }}{% endblocktrans %}</h1>
+    <p><small>Month to date overview for {{ publisher }}.</small></p>
+
+    {% include "adserver/includes/publisher-paid-notice.html" with publisher=publisher only %}
+
+    <div class="row row-cols-1 row-cols-md-4">
+      <div class="col mb-3">
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title text-center">${{ report.total.revenue_share|floatformat:2|intcomma }}</h3>
+            <h6 class="text-muted text-center">{% trans 'Revenue' %}</h6>
+            {% url 'publisher_payouts' publisher.slug as publisher_payout_url %}
+            <p class="text-muted small text-center mb-0">{% blocktrans %}See all your <a href="{{ publisher_payout_url }}">payouts</a>.{% endblocktrans %}</p>
+          </div>
+        </div>
+      </div>
+      <div class="col mb-3">
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title text-center">{{ report.total.ctr|floatformat:3 }}%</h3>
+            <h6 class="text-muted text-center">{% trans 'CTR' %}</h6>
+            <p class="text-muted small text-center mb-0">
+              {% if report.total.ctr >= 0.1 %}
+                <span>{% trans 'Your CTR looks good.' %}</span>
+              {% elif report.total.ctr >= 0.05 %}
+                <span>{% trans 'Your CTR is OK but could use some improvement.' %}</span>
```

Comment:
We should block this on probably having >1000 pageviews in the data, since it seems a bit overly aggressive without many views.

We probably also need to have a document to link folks to here, otherwise it feels weird. I don't know if we have a ton of useful feedback, but we should at least write something up.

Reply:
https://github.com/readthedocs/ethicalads.io/pull/118